README.md

```diff
@@ -78,9 +71,7 @@ interface PngmeSDKParamType {
 }
 ```
 
-The `go` method is an _async_ function that takes eight required parameters.
-The `go` method is the main entrypoint method for invoking the PngmeSdk.
-The `go` method is idempotent, and can be invoked multiple times.
+> The `go` method is idempotent and can be safely invoked multiple times.
```

Comment:
Is there another word besides `idempotent` here? I seriously had to look  that word up.

Reply:
@benfasoli This is the one last one I was curious about.